src/test/scala/nl.knaw.dans.easy.deposit/DepositDirSpec.scala

```diff
@@ -181,19 +182,17 @@ class DepositDirSpec extends TestSupportFixture with MockFactory {
   "writeSplittedDatasetMetadata" should "write 4 files" in {
     val prologue = """<?xml version='1.0' encoding='UTF-8'?>"""
     val message = "Lorum ipsum"
-    val datasetMetadata = DatasetMetadata(
-      privacySensitiveDataPresent = PrivacySensitiveDataPresent.yes,
-      acceptLicenseAgreement = true,
-      messageForDataManager = Some(message)
-    )
+    val datasetMetadata = DatasetMetadata((File("src") / "test" / "resources" / "manual-test" / "datasetmetadata-from-ui-all.json").contentAsString)
```

Comment:
`File("src") / "test" / "resources" / "manual-test" / "datasetmetadata-from-ui-all.json"`
probably the same as 
`File(getClass.getResource("/manual-test/datasetmetadata-from-ui-all.json"))`

(or maybe without the `/` prefix)

same in other places in your tests

Reply:
btw. what manual tests are there...
shouldn't you migrate those to automated tests?

Reply:
These files can be used for `PUT /deposit/{id}/metadata`.

Manual tests should also be tested automatically to make sure they stay valid.

Reply:
friendly reminder for the first comment...